Dockerfile

```diff
@@ -16,6 +16,23 @@ RUN if [ "$DEBUG" = "true" ]; then apt-get update \
 && apt-get install --no-install-recommends -y gdb gdbserver \
 && apt-get clean; fi
 
+WORKDIR "${HOME}"
+RUN mkdir antlr
```

Comment:
Currently, variable HOME is pointing to /home/ubuntu. This antlr folder should get created inside the following path. Not at HOME.
/home/ubuntu/software

Reply:
okay sir, I'll put into this location, "/home/ubuntu/software". but after moving the generated files in to JasmineGraph, we don't need this folder anymore then this home/ubuntu/antlr folder get cleared.

Reply:
sure